client/src/components/Home/index.jsx

```diff
@@ -1,18 +1,27 @@
 import React from 'react'
 import { MostPopularComments } from '../MostPopularComments'
+import { Breakdown } from '../Breakdown'
 import MostPopular from '../../assets/data/MostPopularNewsList'
 import MostComments from '../../assets/data/MostCommentsNewsList'
+import BreakDown from '../../assets/data/BreakdownNewsList'
+import SingleNews from '../../assets/data/SingleNews'
 
 export const Home = () => (
   <div className='home-page-wrapper'>
+    <div className='breakdown-container'>
```

Comment:
Why do we need this wrapper?

Reply:
for interrupting the weights of the selectors